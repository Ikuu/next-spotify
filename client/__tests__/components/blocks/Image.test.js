import React from 'react';
import { create } from 'react-test-renderer';
import Image from '../../../components/blocks/Image';

test('Image renders', () => {
  const tree = create(
    <Image
      alt="Alt"
      naturalHeight={9}
      naturalWidth={16}
      src="www.test.com"
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
