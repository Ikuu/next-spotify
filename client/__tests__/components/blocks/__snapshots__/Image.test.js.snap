// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#ccc",
      "paddingBottom": "56.25%",
      "position": "relative",
    }
  }
>
  <img
    alt="Alt"
    src="www.test.com"
    style={
      Object {
        "left": "50%",
        "position": "absolute",
        "top": "50%",
        "transform": "translate(-50%, -50%)",
        "width": "100%",
      }
    }
  />
</div>
`;
